src/terminal/TerminalWindows/ScrollTerminalWindow/ScrollTerminalWindow.gd

```diff
@@ -0,0 +1,32 @@
+class_name ScrollTerminalWindow
+extends BaseTerminalWindow;
+
+@export_multiline var text : String;
+@export var has_back : bool = true;
+@export var has_exit : bool = false;
+
+var label_scene : PackedScene = preload("res://src/terminal/TerminalWindows/ScrollTerminalWindow/ScrollTerminalWindow.tscn");
+var button_scene : PackedScene = preload("res://src/terminal/TerminalWindows/TerminalButton.tscn");
+
+signal previous_window;
+signal exit_terminal;
+
+func get_menu() -> Node:
+	signals = [previous_window, exit_terminal];
+	var scene_instance : Control = label_scene.instantiate();
+	var label : Label = scene_instance.get_node("ScrollContainer/Label");
```

Comment:
again, you're depending on the scene structure not changing here. the *root* should be handling all of this with whatever minimum data it could need